backend/tracim_backend/tests/functional/test_live_messages.py

```diff
@@ -123,3 +123,39 @@ def test_api__user_live_messages_endpoint_with_GRIP_proxy__ok__user_update__asyn
         assert result["fields"]["author"]
         assert result["event_type"] == "user.modified"
         assert event1.event == "message"
+
+    @pytest.mark.parametrize(
+        "config_section", [{"name": "functional_async_live_test"}], indirect=True
+    )
+    def test_api__user_live_messages_endpoint_with_GRIP_proxy__ok__content_creation__async(
+        self, pushpin, app_config, rq_database_worker
+    ):
+        headers = {"Accept": "text/event-stream"}
```

Comment:
Can you just put a docstring here to clarify what is tested here.

Reply:
I think I will remove this test: I put it here since I saw more "missed" events when creating contents during my firts manual tests, but after more testing this is not especially contents which are concerned (as expected).